pynest/examples/pynest-microcircuit/network.py

```diff
@@ -0,0 +1,347 @@
+                       'overwrite_files': self.sim_dict['overwrite_files'],
+                       }
+        nest.SetKernelStatus(kernel_dict)
+
+    # create neuronal populations
+    def create_populations(self):
+        self.N_ttl = self.net_dict['N_full']
+        self.N_scl = self.net_dict['N_scaling']
+        self.K_scl = self.net_dict['K_scaling']
+        self.synapses = get_total_number_of_synapses()
+        self.synapses_scaled = self.synapses * self.K_scl
+        self.nr_neurons = self.N_ttl * self.N_scl
+        self.K_ext = self.net_dict['K_ext'] * self.K_scl
+        self.w_from_PSP = get_weight(self.net_dict['PSP_e'])
+        self.weight_mat = get_weight(self.net_dict['PSP_mean_matrix'])
+        self.weight_mat_std = self.net_dict['PSP_std_matrix']
+        self.w_ext = self.w_from_PSP
+        self.DC_amp_e = np.zeros(len(self.net_dict['populations']))
+        if self.N_scl != 1:
+            print 'The Number of neurons is scaled by a factor of:'
+            print self.N_scl
+
+        # Scaling of the synapses
+        if self.K_scl != 1:
+            synapses_indegree = self.synapses / (
+                self.N_ttl.reshape(len(self.N_ttl), 1) * self.N_scl)
+            self.weight_mat, self.w_ext, self.DC_amp_e = adj_w_ext_to_K(
+                synapses_indegree,
+                self.K_scl,
+                self.weight_mat, self.w_from_PSP,
+                self.DC_amp_e)
+            print 'The number of synapses is scalled by a factor of:'
+            print self.K_scl
+        else:
+            self.w_ext = self.w_from_PSP
+
+        # Create cortical populations
+        self.pops = []
+        pop_file = open(self.data + 'population_GIDs.dat', 'w+')
+        for i, pop in enumerate(self.net_dict['populations']):
+            population = nest.Create(self.net_dict['neuron_model'],
+                                     int(self.nr_neurons[i]))
+            nest.SetStatus(population,
+                           {'tau_syn_ex': self.net_dict['n_params']['tau_ex'],
+                            'tau_syn_in': self.net_dict['n_params']['tau_in'],
+                            'E_L': self.net_dict['n_params']['E_L'],
+                            'V_th': self.net_dict['n_params']['V_th'],
+                            'V_reset':  self.net_dict['n_params']['V_reset'],
+                            't_ref': self.net_dict['n_params']['t_ref'],
+                            'I_e': self.DC_amp_e[i]
+                            })
+            node_info = nest.GetStatus(population)
+            local_nodes = [(ni['global_id'], ni['vp'])
+                           for ni in node_info if ni['local']]
+            for gid, vp in local_nodes:
+                nest.SetStatus(
+                    [gid],
+                    {'V_m': self.pyrngs[vp].uniform(
+                                self.net_dict['n_params']['V_th'],
+                                self.net_dict['n_params']['V_reset'])})
```

Comment:
Here, the initial membrane potentials are always distributed uniformly, instead of using `V0_mean` and `V0_std`, which are defined `networks_params.py`. Either remove these parameters then, if they are not used anyway, or use them here.
The SLI version uses a normal distribution with customizable mean and sigma, so it would be good to have both versions consistent.